### PR TITLE
Add image to article

### PR DESCRIPTION
**Pull request by davidboll**
#### Description
Changes proposed in this pull request:

* Adds active storage
* Makes it possible to attach an image to an article
#### What I have learned working on this feature:
Working with active storage

#### Screenshots:
skarmavbild 2018-03-21 kl 21 26 50

### CODE REVIEW

File: features/user_can_add_an_image_to_an_article.feature [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  I would like to be able to publish articles with an image
+
+  Background:
+    Given the following categories exist
+      | name    |
+      | Fashion |
+
+  Scenario: Author creates an article
+    Given I am on the "Create Article" page
+    And I fill in "Title" with "David is the best"
```
Comment (jyuw) [inline features/user_can_add_an_image_to_an_article.feature]: Please change to `When`

File: features/user_should_be_able_to_create_articles.feature [hunk 2]
```diff
@@ -13,6 +13,7 @@ Feature: User should be able to create articles
     When I fill in "Title" with "Holger is the best"
     And I fill in "Content" with "Dreamteam for the win!"
     When I select "Fashion" from "Categories"
```
Comment (jyuw) [inline features/user_should_be_able_to_create_articles.feature]: Please change to `And`

**Review by tochman — APPROVED** [review]
Good job!

**Review by jyuw — APPROVED** [review]
Good job guys!

**Pull request merged**